l10n_eu_nace/i18n/hr.po

```diff
@@ -0,0 +1,5003 @@
+# Translation of OpenERP Server.
```

Comment:
This file is broken: https://travis-ci.org/github/OCA/community-data-files/jobs/701976078#L3837-L3866

Please delete it so the bot can continue with the merge.

Reply:
done